src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -149,9 +161,17 @@
 
 (schema/defn ^:always-validate get-jvm-metrics :- JvmMetricsV1
   []
-  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)]
+  (let [runtime-bean (ManagementFactory/getRuntimeMXBean)
+        gc-beans (jmx/mbean-names "java.lang:name=*,type=GarbageCollector")]
     {:heap-memory (jmx/read "java.lang:type=Memory" :HeapMemoryUsage)
      :non-heap-memory (jmx/read "java.lang:type=Memory" :NonHeapMemoryUsage)
+     :file-descriptors (clojure.set/rename-keys
```

Comment:
Nit: we try to avoid using fully qualified namespaces in the code because they can allow situations where tests pass even though you haven't actually done the necessary `require` in the namespace, if it happened to get `required` in some other namespace.  This can lead to ordering issues where a namespace works fine if loaded *after* some other namespace, but doesn't work on its own.  The preferred approach is to add an `:as` clause to your `require` (e.g., `:as set`), and then use that alias in the code.